WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -0,0 +1,171 @@
+<% provide(:title, @competition.name) %>
+
+<% if @admin_view %>
+  <h2>Announcements</h2>
+  <%= link_to "Upload results.", "/results/admin/upload_results.php?competitionId=#{@competition.id}" %>
+  <hr>
+<% end %>
+
+<%= simple_form_for @competition, html: { class: 'form-horizontal are-you-sure no-submit-on-enter', id: 'edit-competition-form' }, wrapper: :horizontal_form,
```

Comment:
I'm getting kicked back to the organizer view when trying to save this form with an error (eg, removing mailto: in a link); this should behave a little better.